### ISSUE TITLE
New accounts cannot enter the game

### ISSUE THREAD

**Issue by Matias-Salvador** [chromiecraft/chromiecraft#2846]
Context:

Hello everyone, I am writing to ask about an error when creating new accounts, I invited a friend to this server, but he could not enter due to an error in the system and this error also happened to me when I tried to create a second account.

Details:

You can create the account on the Chromie web page and you can log in there, but you cannot enter the game (I remark that I have a functional account and it is my second account that does not work, therefore, the client is configured correctly).

On the web when one tries to access the "characters" part, this error appears:

> Fatal error: Uncaught PDOException: SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'ORDER BY COALESCE(`order`, `guid`)' at line 5 in /var/www/html/wp-content/plugins/acore-wp-plugin/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:82 Stack trace: #0 /var/www/html/wp-content/plugins/acore-wp-plugin/vendor/doctrine/dbal/src/Driver/PDO/Connection.php(82): PDO->query('SELECT\n ...') #1 /var/www/html/wp-content/plugins/acore-wp-plugin/vendor/doctrine/dbal/src/Connection.php(1062): Doctrine\DBAL\Driver\PDO\Connection->query('SELECT\n ...') #2 /var/www/html/wp-content/plugins/acore-wp-plugin/src/Components/CharactersMenu/CharactersController.php(31): Doctrine\DBAL\Connection->executeQuery('SELECT\n ...') #3 /var/www/html/wp-content/plugins/acore-wp-plugin/src/Components/CharactersMenu/CharactersMenu.php(34): ACore\Components\CharactersMenu\CharactersController->load in /var/www/html/wp-content/plugins/acore-wp-plugin/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php on line 84
There has been a critical error on this website. Please check your site admin email inbox for instructions.
Learn more about troubleshooting WordPress.

I apologize if this is a known bug and I'm being annoying, but neither my friend nor I have been able to find a solution and it would be a shame for him and other future users to lose interest in the server because of this problem.

In advance, thank you very much for everything.

**Comment by acidmanifesto** [comment]
I just discussed this with the CC staff and they recommend that you try a few things to see if this is a one time issue or not:
- ensure accountname and password do not exceed 16 characters each
- You can create a new account with the same email adress by using an alias

Do let us know what happens on the second try of creation. if the issue still presist do let us know and we will look further into it.

**Comment by Matias-Salvador** [comment]
Both my friend and I were able to solve the problem, we both changed the passwords and everything works perfectly. In my case I use a password manager with "generator" function and therefore I used one of 30 characters; but in the case of my friend he used the default password generated in the Chromie web page and had the same problem (that would be something to examine).

Thank you very much for responding so quickly and having solved the issue, I am delighted with this server and its community, big hug.

Translated with www.DeepL.com/Translator (free version)
